bin/ember-template-lint.js

```diff
@@ -58,35 +60,62 @@ function printErrors(errors) {
 }
 
 function lintFile(linter, filePath, moduleId) {
-  var source = fs.readFileSync(filePath, { encoding: 'utf8' });
+  var source;
+  try {
+    source = rw.readFileSync(filePath, { encoding: 'utf8' });
+  } catch (error) {
+    if (error.code === 'ENXIO') {
```

Comment:
`ENXIO` is the system error code for when we attempt to read from `/dev/stdin` and it is not available.

> “No such device or address.” The system tried to use the device represented by a file you specified, and it couldn’t find the device. This can mean that the device file was installed incorrectly, or that the physical device is missing or not correctly attached to the computer.
> https://www.gnu.org/software/libc/manual/html_node/Error-Codes.html#index-ENXIO

The general user land condition this is handling is when the user has asked to read from stdin but has not provided a stdin pipe. Pretty sure this is the case if they run `ember-template-lint` with no arguments or pass in `/dev/stdin` but stdin is attached to their TTY.

I tried looking further and discovered that this error showed up when [running this test](https://github.com/ember-template-lint/ember-template-lint/blob/4ffbc81077a3582aca4e40b4ae05b9986d8ffcb8/test/unit/bin/ember-template-lint-test.js#L10). because it offers no args the assumption is stdin but nothing has been piped to it so the OS throws an ENXIO error. I added that conditional to get the test to pass. Maybe my understanding is inaccurate?